### PR TITLE
Analytics and Information gathering

### PR DESCRIPTION
**Pull request by GalteXx**

### CODE REVIEW

File: TundraTD/Assets/Scripts/City/CityGates.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ private void OnTriggerEnter(Collider other)
             var mob = other.GetComponent<MobBehaviour>();
             var mobAttack = mob.MobModel.CurrentMobDamage;
             
-            Debug.Log($"{mob.name} attacked tower with {mobAttack} damage");
+            //Debug.Log($"{mob.name} attacked tower with {mobAttack} damage");
```
Comment (TaggedDev) [inline TundraTD/Assets/Scripts/City/CityGates.cs:78]: remove extra debug

File: TundraTD/Assets/Scripts/Mobs/MobsBehaviour/MobBehaviour.cs [hunk 2]
```diff
@@ -54,7 +54,7 @@ protected float TickTimer
 
         public void HitThisMob(float damage, BasicElement damageElement, string sourceName)
         {
-            Debug.Log($"Handling {damage} damage from {sourceName} hitting {name}");
+            //Debug.Log($"Handling {damage} damage from {sourceName} hitting {name}");
```
Comment (TaggedDev) [inline TundraTD/Assets/Scripts/Mobs/MobsBehaviour/MobBehaviour.cs:57]: remove extra debug

**Review by TaggedDev — CHANGES_REQUESTED** [review]

File: TundraTD/Assets/Scripts/Mobs/MobModel.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ public class MobModel : MonoBehaviour
         [SerializeField]
         private SkinnedMeshRenderer skinRenderer;
 
-        [SerializeField] private Rigidbody rigidbody;
+        [SerializeField] new private Rigidbody rigidbody;
```
Comment (IOExcept10n) [inline TundraTD/Assets/Scripts/Mobs/MobModel.cs:29]: I guess there could be something incorrect when using `new` modifier. Maybe there were a variable with the same purpose? If no, you could rename it

**Review by IOExcept10n — APPROVED** [review]

File: TundraTD/Assets/Scripts/Mobs/MobModel.cs (hunk 3 above)
Comment (GalteXx) [reply TundraTD/Assets/Scripts/Mobs/MobModel.cs:29]: Rigidbody is a Obsolete and unused method for all monobehaviours. The use of 'rigitbody' as it is named there is necessary

**Review by TaggedDev — APPROVED** [review]

**Pull request merged**